src/lib/CollectStore.js

```diff
@@ -138,10 +138,17 @@ export default class CollectStore {
       // 8. Creates trigger
       .then(job => {
         connection.job = job
+
         if (!connection.konnector || !connection.konnector.slug) {
           console.error(connection.konnector, 'No konnector slug available to register the new konnector trigger')
           return Promise.reject(new Error('Unexpected error while trying to setup next run of the konnector'))
         }
+
+        if (job.attributes.state === 'queued') {
```

Comment:
All this part seems not necessary, as `this.jobTimeout` seems to just log in the console for debugging purpose.